### PR TITLE
fix: auth copy changes

### PR DESCRIPTION
**Pull request by rebelchris**
### Describe what this PR does
- Cleaned some components as they got a bit mixed up
- Still not happy with how we render modals in normal pages, but for now it was too big of a effort to clean
- Mainly copy changes, and label usages

## Events

Did you introduce any new tracking events?
Don't forget to update the [Analytics Taxonomy sheet](https://docs.google.com/spreadsheets/d/18Lv7zXges9QfVX5VYL1a-Hyl0e1sQ3sLr0OK8YZWKXI/edit#gid=0)

Log the new/changed events below:

| Type   | event_name  | value |
|--------|-------------|-------|
| Change/New | event name  | extra: { ... } |

### On those affected packages:
- [ ] Have you done sanity checks in the webapp?
- [ ] Have you done sanity checks in the extension?
- [ ] Does this not break anything in companion?

### Did you test the modified components media queries?
- [ ] MobileL (420px)
- [ ] Tablet (656px)
- [ ] Laptop (1020px)

#### Did you test on actual mobile devices?
- [ ] iOS (Chrome and Safari)
- [ ] Android

WT-1767 #done

### CODE REVIEW

File: packages/shared/src/components/auth/AuthHeader.tsx [hunk 1]
```diff
@@ -5,17 +5,23 @@ import { CloseModalFunc } from '../modals/common';
...
+function AuthHeader({
+  simplified = false,
   title,
   className,
   onBack,
-}: AuthModalHeaderProps): ReactElement {
+}: AuthHeaderProps): ReactElement {
+  if (simplified) {
+    return <h2 className="font-bold text-center typo-title2">{title}</h2>;
+  }
```
Comment (rebelchris) [inline packages/shared/src/components/auth/AuthHeader.tsx:23]: Basically simplified means outside of the modal, so header needs to be inlined to show per screen :)

File: packages/shared/src/components/auth/LoginForm.tsx [hunk 2]
```diff
@@ -108,6 +111,21 @@ function LoginForm({
           {loginButton}
         </Button>
       </span>
+      {hint && hint === labels.auth.error.invalidEmailOrPassword && (
```
Comment (rebelchris) [inline packages/shared/src/components/auth/LoginForm.tsx:114]: This is a bit of a mission, but the request was to make it more clear so a custom hint exception for this specific hint only

File: packages/webapp/pages/onboarding.tsx [hunk 3]
```diff
@@ -163,6 +161,65 @@ export function OnboardPage(): ReactElement {
     setHasSelectTopics(hasTopics);
   };
 
+  const getContent = (): ReactElement => {
```
Comment (rebelchris) [inline packages/webapp/pages/onboarding.tsx:164]: Wasn't sure why this was "conditionalWrapped" it should have been conditionally rendered as adjusted.

Comment (sshanzel) [reply packages/webapp/pages/onboarding.tsx:164]: I think the conditional wrap came from the requirement of auth options having the custom `title` and `content` as opposed to having just the `AuthOptions` now.

**Review by omBratteng — APPROVED** [review]
Looks good to me, also simplifies some of the code I'm currently working on.

**Comment by omBratteng** [comment]
Should this copy also be updated?
https://github.com/dailydotdev/apps/blob/9bd26c30cf9246327f3e99d7bed1c2999dad38f0/packages/shared/src/components/onboarding/MemberAlready.tsx#L21

**Comment by rebelchris** [comment]
@omBratteng The already member one is for a seperate task for now, @sshanzel  is working on that one.
It's basically a complete revamp of how it looks and works

File: packages/shared/src/components/auth/AuthModalFooter.tsx [hunk 4]
```diff
@@ -1,33 +1,35 @@
...
-        {isLogin ? 'Don’t have an account?' : 'Already have an account?'}
-      </Modal.Text>
+      {text.body && <Modal.Text>{text.body}</Modal.Text>}
       <ClickableText
         className="ml-1 underline text-theme-label-primary"
-        onClick={() => onIsLogin(!isLogin)}
+        onClick={onClick}
       >
-        {isLogin ? 'Sign up' : 'Log in'}
+        {text.button}
```
Comment (DragosIorgulescu) [inline packages/shared/src/components/auth/AuthModalFooter.tsx:32]: this looks funky, what is the difference between `text.body` and `text.button` here?

File: packages/shared/src/components/auth/LoginForm.tsx (hunk 2 above)
Comment (DragosIorgulescu) [reply packages/shared/src/components/auth/LoginForm.tsx:114]: assuming `labels.auth.error.invalidEmailOrPass` is never null or undefined, I think there would be no need to check if `hint` exists as well
```suggestion
      {hint === labels.auth.error.invalidEmailOrPassword && (
```

**Review by DragosIorgulescu — APPROVED** [review]
solid work 👌  some very minor questions from me -- looks like there is a failing spec as well but not sure if it's just a flaky one

File: packages/shared/src/components/auth/AuthModalFooter.tsx (hunk 4 above)
Comment (rebelchris) [reply packages/shared/src/components/auth/AuthModalFooter.tsx:32]: We render a button which has some text in it, and some descriptive optional text on top.

**Pull request merged**